bauhaus/main.py

```diff
@@ -5,6 +5,15 @@
 from bauhaus.workflows import availableWorkflows
 from bauhaus.utils import mkdirp
 
+def doWorkflowHelp(args):
+    if not args.workflow :
+        print "Please pick a workflow with the --workflow option:"
+        print ' '.join(availableWorkflows.keys())
+        return
+
+    wfg = availableWorkflows[args.workflow]()
+    wfg.help()
```

Comment:
Nice thing about run-time python is that if no one uses this feature, we don't need to implement it everywhere.